.env.example

```
@@ -1,47 +1,71 @@
-FEATURE_CLAMAV_ENABLED=true
-FEATURE_GATED_CONTENT_ENABLED=true
+# API Rate Limiting
+RATE_LIMIT_TTL=60000  # 1 minute
+RATE_LIMIT_MAX=100    # 100 requests per minute
 
-# Environment
-NODE_ENV=development
-PORT=3000
+# Monitoring
+PROMETHEUS_ENABLED=true
+SENTRY_DSN=  # Optional error tracking
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)